From: Qiang Ma <maqianga@uniontech.com>
Subject: arm64: kexec: adjust the debug print of kexec_image_info
Date: Fri, 19 Dec 2025 17:31:34 +0800

In the previous patch, we move start/head of kimage to the generic code,
so, we remove them to avoid duplicate printing.

Link: https://lkml.kernel.org/r/20251219093134.2268620-5-maqianga@uniontech.com
Signed-off-by: Qiang Ma <maqianga@uniontech.com>
Acked-by: Will Deacon <will@kernel.org>
Cc: Baoquan He <bhe@redhat.com>
Cc: Catalin Marinas <catalin.marinas@arm.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 arch/arm64/kernel/machine_kexec.c |    2 --
 1 file changed, 2 deletions(-)

--- a/arch/arm64/kernel/machine_kexec.c~arm64-kexec-adjust-the-debug-print-of-kexec_image_info
+++ a/arch/arm64/kernel/machine_kexec.c
@@ -34,8 +34,6 @@ static void _kexec_image_info(const char
 {
 	kexec_dprintk("%s:%d:\n", func, line);
 	kexec_dprintk("  kexec kimage info:\n");
-	kexec_dprintk("    type:        %d\n", kimage->type);
-	kexec_dprintk("    head:        %lx\n", kimage->head);
 	kexec_dprintk("    kern_reloc: %pa\n", &kimage->arch.kern_reloc);
 	kexec_dprintk("    el2_vectors: %pa\n", &kimage->arch.el2_vectors);
 }
_
